src/node.cc

```diff
@@ -156,6 +157,38 @@ static Isolate* node_isolate = nullptr;
 static v8::Platform* default_platform;
 
 
+static void PrintErrorString(const char* format, ...) {
+  va_list ap;
+  va_start(ap, format);
+#ifdef _WIN32
+  HANDLE stderr_handle = GetStdHandle(STD_ERROR_HANDLE);
+
+  // Check if stderr is something other than a tty/console
+  if (stderr_handle == INVALID_HANDLE_VALUE ||
+      stderr_handle == nullptr ||
+      uv_guess_handle(_fileno(stderr)) != UV_TTY) {
+    vfprintf(stderr, format, ap);
+    return;
+  }
+
+  // Fill in any placeholders
+  size_t n = vsnprintf(nullptr, 0, format, ap);
```

Comment:
I'm kind of surprised this works, I thought vsnprintf() on Windows returned -1 when the output buffer is too small.